### PR TITLE
updated License_badge

### LINKED ISSUE #185: Ensure that the code samples are correct
A lot of stories feature repetitive code. To avoid duplication we generally use functions like
```js
export const heading = (level) => `<h${level}>Heading ${level}</h${level}>`
```
and then we write stories like 
```html
<Preview>
  <Story>{heading(1)}</Story>
</Preview>
```

This eliminates the code repetition but leads to incorrect code previews.
Screen Shot 2020-03-18 at 11 45 35

The correct code can be inserted into the preview using the `mdxSource` prop on the `Preview` component. So a better story would look like
```html
<Preview mdxSource={heading(1)}>
  <Story>{heading(1)}</Story>
</Preview>
```
An example can be found in PR #176.

The goal is to have this done across all stories in the storybook where such function-based code is used.

### PR DESCRIPTION
**Pull request by megha070**
**Fixes**

Fixes #185  by @dhruvkb 

**Description**
Update license_badge.stories.mdx file so as to get correct code Previews

**Type of PR**
This PR is a hotfix.

**Technicalities**
Made use of mdxsource prop so as to get correct code sample

**Tests**

**Screenshots**

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Can't merge till #209 is merged.

**Comment by dhruvkb** [comment]
@megha070 this is unblocked now, but still has conflicts.

**Comment by megha070** [comment]
image
Ran it locally! It is working fine. Could you please tell me how do I resolve these?

File: src/stories/license_badge.stories.mdx [hunk 1]
```diff
@@ -22,85 +23,93 @@ export const licenseSVG = (size, license) => {
 </svg>`
 }
 
-<Preview>
+<Preview mdxSource={licenseSVG("big", "by"),licenseSVG("small", "by")}>
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: I'm not sure you can write comma separated `mdxSource`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
This seems wrong. Check if the MDX source code appears correctly under the badges.

File: src/stories/license_badge.stories.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: Don't replace `'` with `"`.

File: src/stories/license_badge.stories.mdx [hunk 3]
```diff
@@ -10,6 +10,7 @@ CC BY is represented by the CC logo and the icon of a person. Similar icons exis
 
 ## Licenses
 
+
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: Why?

File: src/stories/license_badge.stories.mdx [hunk 4]
```diff
@@ -22,85 +23,94 @@ export const licenseSVG = (size, license) => {
...
   <Story name="BY" height="96px" parameters={{
-    design: figmaConfig("574%3A111")
-  }}>{`
-    <div>${licenseSVG('big', 'by')}</div>
-    <div>${licenseSVG('small', 'by')}</div>
+      design: figmaConfig("574%3A111")
+    }}
+  >{`
+    <div>${licenseSVG("big", "by")}</div>
+    <div>${licenseSVG("small", "by")}</div>
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: This change is meaningless.

File: src/stories/license_badge.stories.mdx [hunk 5]
```diff
@@ -22,85 +23,94 @@ export const licenseSVG = (size, license) => {
 </svg>`
 }
 
-<Preview>
+<Preview mdxSource={`${licenseSVG("big", "by")}\n${licenseSVG("small", "by")}`}>
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: Use single quotes.
```suggestion
<Preview mdxSource={`${licenseSVG('big', 'by')}\n${licenseSVG('small', 'by')}`}>
```

File: src/stories/license_badge.stories.mdx [hunk 6]
```diff
@@ -120,7 +130,7 @@ group from within the licenses.
 <svg width="120px" height="42px">
   <!-- Big sized CC BY license badge -->
   <use
-    href="@creativecommons/vocabulary/assets/license_badges/big/by.svg#layer1">
+     href="@creativecommons/vocabulary/assets/license_badges/big/by.svg#layer1">
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: Remove the stray space.

File: src/stories/license_badge.stories.mdx [hunk 7]
```diff
@@ -22,85 +23,94 @@ export const licenseSVG = (size, license) => {
...
-  }}>{`
-    <div>${licenseSVG('big', 'by')}</div>
-    <div>${licenseSVG('small', 'by')}</div>
+      design: figmaConfig("574%3A111")
+    }}
+  >{`
+    <div>${licenseSVG("big", "by")}</div>
+    <div>${licenseSVG("small", "by")}</div>
   `}</Story>
 </Preview>
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx:33]: The above feedback applies to all stories below as well.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: src/stories/license_badge.stories.mdx [hunk 8]
```diff
@@ -1,4 +1,4 @@
-import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: Drop the semicolon at the end.

File: src/stories/license_badge.stories.mdx [hunk 9]
```diff
@@ -21,86 +21,93 @@ export const licenseSVG = (size, license) => {
...
   <Story name="BY" height="96px" parameters={{
-    design: figmaConfig("574%3A111")
-  }}>{`
-    <div>${licenseSVG('big', 'by')}</div>
-    <div>${licenseSVG('small', 'by')}</div>
+      design: figmaConfig("574%3A111")
+    }}
+  >{`
+    <div>${licenseSVG('big','by')}</div>
+    <div>${licenseSVG('small','by')}</div>
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: These lines should not be appearing as changed. Make them the same as they were before.

File: src/stories/license_badge.stories.mdx [hunk 10]
```diff
@@ -119,14 +126,12 @@ group from within the licenses.
 ```html
 <svg width="120px" height="42px">
   <!-- Big sized CC BY license badge -->
-  <use
-    href="@creativecommons/vocabulary/assets/license_badges/big/by.svg#layer1">
+  <use href="@creativecommons/vocabulary/assets/license_badges/big/by.svg#layer1">
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: This change should be undone.

File: src/stories/license_badge.stories.mdx [hunk 11]
```diff
@@ -119,14 +126,12 @@ group from within the licenses.
...
   </use>
 </svg>
 <svg width="80px" height="15px">
   <!-- Small sized CC BY license badge -->
-  <use
-    href="@creativecommons/vocabulary/assets/license_badges/small/by.svg#layer1">
+  <use href="@creativecommons/vocabulary/assets/license_badges/small/by.svg#layer1">
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.mdx]: This change should be undone.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**